### PR TITLE
Update Opta AWS doc

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

Content clean-up, fixed broken links.

Context: https://flyte-org.slack.com/archives/C02PN54V5FH/p1639375332006200 & https://flyte-org.slack.com/archives/C02PN54V5FH/p1639308124004500?thread_ts=1639294659.003500&cid=C02PN54V5FH

### CODE REVIEW

File: rsts/deployment/aws/opta.rst [hunk 1]
```diff
@@ -1,130 +1,155 @@
...
-Flyte uses SQS to power its task scheduling system, and flyte.yaml creates said queue (via the schedulesQueue
-module) with encryption and principle of least privilege rbac access like the other SQS queue above.
+Production-grade Queueing System
+********************************
+Flyte uses SQS to power its task scheduling system, and ``flyte.yaml`` creates said queue (via the schedulesQueue
+module) with encryption and principle of least privilege RBAC access like the SQS queue mentioned above.
 
-**Secure IAM Roles for Data and Control Planes**
+Secure IAM Roles for Data and Control Planes
+********************************************
```
Comment (samhita-alla) [inline rsts/deployment/aws/opta.rst:102]: What do we have to write here? Currently, it's empty.

File: rsts/deployment/aws/opta.rst [hunk 2]
```diff
@@ -1,130 +1,155 @@
...
 
 
-**Flyte Deployment via Helm**
-A Flyte deployment contains around 50 kubernetes resources.
+Flyte Deployment via Helm
+*************************
+Flyte deployment contains around 50 Kubernetes resources.
```
Comment (samhita-alla) [inline rsts/deployment/aws/opta.rst:107]: Is this correct, or can we remove it?

File: rsts/deployment/aws/opta.rst (hunk 1 above)
Comment (pmahindrakar-oss) [reply rsts/deployment/aws/opta.rst:102]: Adding @wild-endeavor

File: rsts/deployment/aws/opta.rst (hunk 2 above)
Comment (pmahindrakar-oss) [reply rsts/deployment/aws/opta.rst:107]: Sound about right.
```
(base) ➜  flyte git:(master) ✗ kubectl get pods -n flyte |wc -l                                     
      15
(base) ➜  flyte git:(master) ✗ kubectl get deployments -n flyte |wc -l
      11
(base) ➜  flyte git:(master) ✗ kubectl get sa -n flyte |wc -l
      10
(base) ➜  flyte git:(master) ✗ kubectl get service -n flyte |wc -l 
      10
(base) ➜  flyte git:(master) ✗ kubectl get configmap -n flyte |wc -l
      11
```

**Review by pmahindrakar-oss — DISMISSED** [review]

File: rsts/deployment/aws/opta.rst (hunk 2 above)
Comment (samhita-alla) [reply rsts/deployment/aws/opta.rst:107]: Should this be a part of the Opta YAMLs section?

Comment (pmahindrakar-oss) [reply rsts/deployment/aws/opta.rst:107]: Not required according to me

**Review by evalsocket — DISMISSED** [review]

**Comment by samhita-alla** [comment]
@evalsocket / @pmahindrakar-oss +1 again, please.

File: rsts/deployment/aws/opta.rst (hunk 1 above)
Comment (samhita-alla) [reply rsts/deployment/aws/opta.rst:102]: Added some content for now.

**Review by evalsocket — APPROVED** [review]

**Pull request merged**